lib/RestructureCFG/DAGifyPass.cpp

```diff
@@ -21,7 +21,8 @@ using namespace llvm;
 Logger<> DAGifyPassLogger("dagify");
 
 /// Helper function which transform a retreating edge into a `goto` edge
-static void processRetreating(const revng::detail::EdgeDescriptor<BasicBlock *>
+static void processRetreating(ScopeGraphBuilder &SGBuilder,
```

Comment:
Instead of adding an argument, just make this a method of `DAGifyPassImpl`.

Reply:
I made this a method of `DAGifyPassImpl` here https://github.com/revng/revng/blob/3a9379ef740d1d2b9f7bcad02cc962719251e02a/lib/RestructureCFG/DAGifyPass.cpp#L32